### PR TITLE
Email settings combine feature

### PR DESCRIPTION
**Pull request by cjlewis11**
Removed the virtual enviorment binaries.
Tested with email verification on signup process.

### CODE REVIEW

File: etabotsite/etabotsite/settings.py [hunk 1]
```diff
@@ -266,15 +267,7 @@
...
-sys_email_settings = {}
-try:
-    with open('sys_email_settings.json') as f:
-        sys_email_settings = json.load(f)
-    logging.info('loaded sys_email_settings.json: {}'.format(
-        sys_email_settings.keys()))
-except Exception as e:
-    logging.warning('Cannot load sys_email_settings due to "{}". \
-Will use default values'.format(e))
+sys_email_settings = custom_settings.get('SYS_EMAIL_SETTINGS')
```
Comment (aradnaev) [inline etabotsite/etabotsite/settings.py]: what would happen if custom_settings doesn't have this key?
please accommodate for this use case

File: etabotsite/etabotsite/settings.py [hunk 2]
```diff
@@ -27,6 +27,7 @@
 PLATFORM = platform.system()
 logging.info("PLATFORM={}".format(PLATFORM))
 LOCAL_MODE = (PLATFORM == 'Darwin')
+#LOCAL_MODE = True
```
Comment (aradnaev) [inline etabotsite/etabotsite/settings.py]: please remove the unnecessary comment

Comment (cjlewis11) [reply etabotsite/etabotsite/settings.py]: Removed it

File: etabotsite/etabotsite/settings.py (hunk 1 above)
Comment (cjlewis11) [reply etabotsite/etabotsite/settings.py]: Added if to check if the key existed in the custom settings, if not we print a warning. If it is in the custom settings we load all the information. 

Also added a check for the AUTHLIB to spit out warning if there is an issue.

**Comment by aradnaev** [comment]
@cjlewis11 , please remove *.pyc files

**Comment by aradnaev** [comment]
@cjlewis11 , please add the check for production we discussed (raise error if email settings are missing in prod mode)

**Comment by aradnaev** [comment]
tested OK, thanks @cjlewis11

**Pull request merged**